solutions/observability/apm/data-stream-routing.md

```diff
@@ -0,0 +1,18 @@
+---
+mapped_pages:
+applies_to:
+  stack:
+  serverless:
+---
+
+# Data stream routing [apm-open-telemetry-data-stream-routing]
+
+Elastic APM supports [routing APM data](/solutions/observability/apm/data-streams.md#apm-data-stream-rerouting) to user-defined data stream names using the [`reroute` processor](elasticsearch://reference/enrich-processor/reroute-processor.md). However, you can also ingest OTLP data without having to create new ingest pipelines.
+
+## Setting data stream attributes
+
+To automatically route OTLP data, set the `data_stream.dataset` and `data_stream.namespace` attributes. These attributes map to the respective [ECS fields](ecs://reference/ecs-data_stream.md).
+
+You can set the `data_stream` attributes at resource level, scope level, span level, or span event level. Elastic parses the attributes in increasing order of precedence. For example, span event `data_stream` attributes override the span `data_stream` attributes. This implies that `data_stream` attributes are inherited from previous levels. If a span doesn't specify `data_stream` attributes, it uses the scope attributes.
```

Comment:
> resource level, scope level, span level, or span event level

Not entirely accurate. It works at resource level, scope level, and record level (span, span event, log record, data point)

Reply:
@carsonip Feel free to open a small patch and I'll merge